modular_skyrat/modules/modular_reagents/code/reagents/reagent_containers/syringes.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/reagent_containers/syringe/ataraxydone
+	name = "syringe (ataraxydone)"
+	desc = "Contains ataraxydone, a wondrous stasis-inducing chemical usable on both the living and dead. The label warns that, while this chemical may allow for seemingly-inexplicable resistance to a variety of temporal and environmental effects, it impedes activity, and its byproducts cause significant genetic damage."
+	list_reagents = list(/datum/reagent/medicine/ataraxydone = 15)
```

Comment:
Will it also contain the impure chem in it? If so, should probably inform people on the syringe that its cellular damage will need to be dealt with after use, or something like that.